src/modules/solution-explorer/solution-explorer-solution/delete-diagram-modal/delete-diagram-modal.html

```diff
@@ -0,0 +1,18 @@
+<template>
+  <div if.bind="showModal">
+    <div class="modal show show-modal" tabindex="-1" role="dialog" id="deleteDiagramModal">
+      <div class="modal-dialog delete-diagram-modal" role="document">
+        <div class="modal-content">
+          <div class="modal-header">
+            <h4 class="modal-title">Do you want to delete this diagram?</h4>
```

Comment:
Maybe put a little more emphasis on this?
This operation is rather permanent, after all.
```suggestion
            <h4 class="modal-title">Are you sure you want to delete this diagram?</h4>
```